src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -269,6 +269,23 @@ else if (flutterLaunchMode == FlutterLaunchMode.RELEASE) {
     return new FlutterCommand(this, root.getRoot(), FlutterCommand.Type.ATTACH, args.toArray(new String[]{ }));
   }
 
+  public FlutterCommand flutterRunWeb(@NotNull PubRoot root, @NotNull RunMode mode, boolean hotReload) {
+    // flutter packages pub global run webdev serve [--debug] [--hot-reload]
+    final List<String> args = new ArrayList<>();
+    args.add("global");
+    args.add("run");
+    args.add("webdev");
+    args.add("daemon");
+    if (mode == RunMode.DEBUG) {
+      //args.add("--debug");
```

Comment:
do these need to be commented out? I imagine they might apply if/when we support debug mode.